app/gui2/src/components/GraphEditor/widgets/WidgetSelection.vue

```diff
@@ -100,23 +105,26 @@ const selectedTag = computed(() => {
   }
 })
 
-const selectedExpression = computed(() => {
-  if (selectedTag.value == null) return WidgetInput.valueRepr(props.input)
-  return selectedTag.value.expression
+const selectedLabel = computed(() => {
+  return selectedTag.value?.label
 })
 const innerWidgetInput = computed(() => {
-  if (selectedTag.value == null) return props.input
-  const parameters = selectedTag.value.parameters
-  if (!parameters) return props.input
-  const config = functionCallConfiguration(parameters)
-  return { ...props.input, dynamicConfig: config }
+  if (props.input.dynamicConfig == null) return props.input
```

Comment:
Two early return branches here could be easily combined into one by using the null chain operator.

Reply:
Does not help much, because `config` variable is needed down the line.